script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -48,30 +48,30 @@ I won't be able to defeat Vlov with my strength tonight, but I can at least put
 "What do you mean 'hinder his movement'? Don't tell me that you're going to grab him and pin him down from above... are you?"
 <Page15>
 "I'm not going to do anything that uncivilized.
-It would be futile to hold him down directly. Besides, I've never done it before. It would be totally unnecessary."
+It would be futile to hold him down directly. Besides, I've never done it before, and I don't feel like starting now."
 <Page16>
 "I-I see. In that case, how are you going to subdue him?"
 "It depends on the location, but essentially I'll be using chains."
 <Page17>
-"Chains...? By chains, you mean those chains?"
+"Chains...? By chains, you mean those chains?"//He says あの鎖 but have we seen any chains to reference? Or is he just saying 'you mean, plain old chains'?
```

Comment:
I don't think the chains in question have been referenced or seen up to this point. I think he just means "Do you just mean normal chains" as in "is that all?"